src/js/Controllers/UIController.js

```diff
@@ -213,6 +262,13 @@ class UIController {
                     this.listener.send(RpcFactory.AlertAbortedResponse(app.alert.msgID))
                     store.dispatch(closeAlert(app.alert.msgID, rpc.params.appID))
                     return true
+                } else if (rpc.params.functionID === 64 && app.alert.showAlert
+                    && (rpc.params.cancelID === undefined || rpc.params.cancelID === app.alert.cancelID)) {
+                   clearTimeout(this.timers[app.alert.msgID])
+                   delete this.timers[app.alert.msgID]
+                   this.listener.send(RpcFactory.SubtleAlertErrorResponse(app.alert.msgID, 5, 'subtle alert was cancelled'))
```

Comment:
Please use a constant or add a note stating that result code 5 refers to `ABORTED`

Reply:
Marking as resolved for this PR. Please refer to https://github.com/smartdevicelink/generic_hmi/pull/287#discussion_r469189664